pkg/inject/inject.go

```diff
@@ -1175,14 +1176,53 @@ func ToWholeCPUCores(q k8sResource.Quantity) (int64, error) {
 	return 0, fmt.Errorf("Could not parse cores: %s", q.String())
 }
 
+// getPodInboundPorts will return a string-formatted list of ports based on a
+// PodSpec object. The function will check each container in the pod and extract
+// any defined ports. Additionally, it will also extract any healthcheck target
+// probes, provided the probe is an HTTP healthcheck
 func getPodInboundPorts(podSpec *corev1.PodSpec) string {
+	uniquePorts := make(map[int32]struct{})
```

Comment:
TIOLI

another approach would be to build up the ports at a set: `ports := make(map[int32]struct{})` and then at the end extract the keys to a slice and sort it.  This is conceptually simpler imo because it only uses one variable (ports) instead of needing a second variable to track uniqueness.  The downside is that this doesn't preserve the order of the container ports, but this probably isn't important.

Reply:
Yeah looks much better, made the change and used `sort.Strings` to sort the ports 👍🏻